WIP: add standard names and better long names

meggart opened the pull request:
I started this branch to add correct standard names and better long names to the variable descriptions. I also started to include a url to the original source provider. Is this a good idea? SHould we add more info about the original data here (project name, dataset version) or should this be done in the PH?

gunbra32 commented:
The PH should retrieve as much metadata as possible automatically from the input data or the data cube. Thus, I strongly opt for adding metadata that is not already present in the input files during cube generation via the data providers.

meggart commented:
Thanks Gunnar,  I agree. I can not work on this until Thursday but will continue afterwards.

meggart commented:
Ok, I have updated the PR and added some references as well as a short description of the dataset that we might use for dataset description. Once I am at it, can I also add `plot_min` and `plot_max` attributes which we could use for simpler plotting or is this not the appropriate place?
